<launch>
    <node pkg="robo_control" type="control_strategy_3_node" name="control_strategy_3_node" output="screen">
    <!-- 蹲点的位置 -->
    <rosparam param="pointA"> [2.6, 3.1, 0]</rosparam>  <!-- 2 车 1 点 -->
    <rosparam param="pointB"> [4.0, 4.2, 0]</rosparam>  
    <rosparam param="pointC"> [1.3, 1.8, 0]</rosparam>  <!-- 2 车 2 点 -->
    <rosparam param="pointD"> [2.6, 0.8, 0]</rosparam>  <!-- 1 车 2 点 -->
    <rosparam param="pointE"> [2.6, 2.0, 0]</rosparam>  <!-- 1 车 1 点 -->

    <!-- 2 车 -->
    <rosparam param="robo_point1"> [2.6, 3.1, 0]</rosparam>
    <rosparam param="robo_point2"> [1.3, 1.8, 0]</rosparam>

    </node>
</launch>
<!--  -->
